### ISSUE TITLE
YAML empty objects get sripped away 

### ISSUE THREAD

**Issue by a-waider** [thim81/openapi-format#72]
I've experienced that empty objects in the yaml gets stripped away. An example for a request body in openapi with an epmty object gets stripped away.

E.g.
``` yaml
example:
  default:
    key1: "value1"
    key: {}
```

becomes

``` yaml
example:
  default:
    key1: "value1"
```

Is this related to the parsing library used in openapi-format? Do you have any suggestions how to preserve the empty object?

**Comment by a-waider** [comment]
So, I found the location responsible for removing empty objects: [openapi-format.js#L443](https://github.com/thim81/openapi-format/blob/main/openapi-format.js#L443). When commenting out, it works as intended.

What are the reasons to remove empty objects? Can we just delete lines 440 to 444 in openapi-format.js?

**Comment by a-waider** [comment]
In my use case there is an example for a request body which contains a key with an empty map.

**Comment by a-waider** [comment]
That would be awesome

**Comment by a-waider** [comment]
Any news on this issue?

**Comment by thim81** [comment]
hi @a-waider 
There is a fix ready #73
If the empty object is in a default example, it gets skipped from being deleted.
Would that solve you case?

**Comment by thim81** [comment]
hi @a-waider 

I improved the logic to delete empty objects, like you suggested and released a new version 1.13.1 of openapi-format.
Let me know, if it behaves as you expect.

**Comment by a-waider** [comment]
Thanks for you effort. The new version solves this issue :)
